components/Request/Request.jsx

```diff
@@ -8,6 +8,7 @@ import { REDUCER_KEY } from '../../reducers';
 
 const Request = ({ error, matchId, loading, progress, dispatchRequest, dispatchMatchId }) => {
   function submit() {
+    document.getElementById('match_id').value = '';
```

Comment:
That was my impression, granted I am new to it so maybe I'm likely missing something crucial here, but I could not get that to work. dispatchMatchId did not clear the text upon submission. I could just remove the line entirely until I figure out something better. Alternatively, one could keep the matchId in the TextField and prevent the floatinglabel from moving downwards? But you would lose the luster of the animation, so I do not like that as much.